large_image/tilesource/base.py

```diff
@@ -202,6 +203,27 @@ def _setStyle(self, style):
     def style(self):
         return self._style
 
+    @property
+    def dtype(self):
+        if not self._dtype:
+            with self._styleLock:
+                if not hasattr(self, '_skipStyle'):
+                    self._setSkipStyle(True)
+                    try:
+                        sample, format = self.getRegion(
```

Comment:
This should be
```
sample, format = self.getRegion(
                            region=dict(left=0, top=0, width=1, height=1),
                            format=TILE_FORMAT_NUMPY)
```
Setting `left` and `right` both to zero would get a zero-width result which might invoke a quick response with no data and therefore some default dtype (probably uint8); we want `left` and `top`.  The region endpoint uses `regionWidth` and `regionHeight`, but the actual region API call uses `width` and `height` as a property of the region parameter.  Specifying the output width and height would be done via `output=dict(maxWidth=1, maxHeight=1)`, but is unnecessary since we are just getting one pixel anyway.

Reply:
Fixed in [e015def](https://github.com/girder/large_image/pull/1144/commits/e015deff103cedccef31b6f1ed9d634240e2b2ae)